src/app/GitCommands/Git/GitModule.cs

```diff
@@ -97,7 +97,7 @@ public GitModule(string? workingDir)
             _wslDistro = AppSettings.WslGitEnabled ? PathUtil.GetWslDistro(WorkingDir) : "";
             if (!string.IsNullOrEmpty(_wslDistro))
             {
-                _gitExecutable = new Executable(() => AppSettings.WslGitCommand, WorkingDir, $"-d {_wslDistro} {AppSettings.WslGitPath} ");
+                _gitExecutable = new Executable(() => AppSettings.WslGitCommand, WorkingDir, $"-d {_wslDistro} --cd {WorkingDir.Quote()} {AppSettings.WslGitPath} ");
```

Comment:
Remove the trailing '\' in WorkingDir
It works in Windows it seems but look suspisious to me... Pasting in unix like terminals fail
```suggestion
                _gitExecutable = new Executable(() => AppSettings.WslGitCommand, WorkingDir, $"-d {_wslDistro} --cd {WorkingDir.TrimEdd('\\').Quote()} {AppSettings.WslGitPath} ");
```

Reply:
Should the path be in POSIX though? And, thus, the trailing backslash to be removed be `/`?